### DISCUSSION TITLE
删除笔记时提示403

### DISCUSSION THREAD

**Discussion by xingyunxiaofei** [Tokinx/Wing#146] [问答]
删除笔记或者修改笔记的时候提示403，在这之前尝试过将主题目录权限设置为777，依然无济于事
![QQ截图20240429100827](https://github.com/Tokinx/Wing/assets/168389602/6501404e-7079-4349-b61e-2e77acc8bbdc)

**Comment by xingyunxiaofei** [comment]
哥哥姐姐们帮忙解决下这个问题，感谢！

**Comment by Tokinx** [comment]
看起来像是被拦截了，但是仅凭图片无法进一步判断
